uber/templates/regform.html

```diff
@@ -91,6 +91,14 @@
         });
     {% endif %}
 
+    $(function () {
+        staffingClicked();
+        if ($.field('staffing')) {
+            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
```

Comment:
I considered that. Unchecking `staffing` will automatically remove all of an attendee's shifts. I wasn't sure if we want to destructively change things without warning the admin before hand, so this forces them to manually remove shifts.

On the other hand, it's a super convenient way of removing someone's shifts if that is what is really intended.